qscintilla2/bld.bat

```diff
@@ -0,0 +1,63 @@
+@echo off
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+if "%PY_VER%"=="2.7" (
+	set VC9DIR=%LOCALAPPDATA%\Programs\Common\Microsoft\Visual C++ for Python\9.0
+	IF EXIST !VC9DIR! ( 
+		set PATH=!VC9DIR!;"%PATH%"
+		if %ARCH% == 32 call vcvarsall.bat x86
+		if %ARCH% == 64 call vcvarsall.bat amd64
+		:: vcvarsall does something strange with PATH, preventing xcopy to be found later. Fix this
+		set PATH=!PATH!;%SYSTEMROOT%\system32
+	)
+)
+if errorlevel 1 exit 1
+
+:: Set QMAKESPEC to the appropriate MSVC compiler
+set QMAKESPEC=%LIBRARY_PREFIX%\mkspecs\win32-msvc-default
+
+@echo ====================================================
+@echo Building Qscintilla2
+@echo ====================================================
+:: Go to the source folder and enter the Qt4Qt5 dir
+cd %SRC_DIR%\Qt4Qt5
+:: Use qmake to generate a make file
+%LIBRARY_BIN%\qmake qscintilla.pro
```

Comment:
Also `qmake` binaries will be renamed eventually, but this change could take more time than the OSX one :-)